spec/duration.html

```diff
@@ -1150,16 +1155,16 @@ <h1>
           _milliseconds_: an integer,
           _microseconds_: an integer,
           _nanoseconds_: an integer,
-          _offsetShift_: an integer,
-        )
+        ): an integer
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It computes an integer number of nanoseconds from the given units, applying a given time zone offset shift in nanoseconds when converting from days to hours.</dd>
+        <dd>
+          It computes an integer number of nanoseconds from the given units.
+          Note that this operation should not be used with _days_ &ne; 0 if the calculation is relative to a Temporal.ZonedDateTime.
```

Comment:
This un-enforceable note makes me a bit nervous. Seems easy to miss this requirement upstream. 

Maybe this AO should not take a _days_ parameter at all, and instead require the caller to do the `days × 24 + hours`  math into the `_hours_` parameter? This might make it more likely that no ZDT-relative days are introduced.

Or it might be fine as-is. Your call.

Reply:
I like this idea, thanks for the suggestion.